cmd/diff-main.go

```diff
@@ -174,7 +176,7 @@ func doDiffMain(firstURL, secondURL string) error {
 	}
 
 	// Diff first and second urls.
-	for diffMsg := range objectDifference(firstClient, secondClient, firstURL, secondURL) {
+	for diffMsg := range objectDifference(firstClient, secondClient, firstURL, secondURL, false) {
```

Comment:
Is this an intended false here ? (to avoid breaking the existing deployments ?)

Reply:
Yes it's meaningless for `diff` command